test/parallel/test-tls-connect.js

```diff
@@ -37,7 +37,7 @@ const path = require('path');
   const cert = fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'));
   const key = fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem'));
 
-  const options = {cert: cert, key: key, port: common.PORT};
+  const options = {cert: cert, key: key, port: 0};
   const conn = tls.connect(options, common.mustNotCall());
 
   conn.on('error', common.mustCall());
```

Comment:
Using `port: 0` I got the same on Linux, however the error code changes to `EADDRNOTAVAIL` on OS X for some reason.